chain/client/src/rocksdb_metrics.rs

```diff
@@ -0,0 +1,151 @@
+use near_metrics::{try_create_gauge_vec, try_create_int_gauge};
+use near_store::db::{StatsValue, StoreStatistics};
+use once_cell::sync::Lazy;
+use prometheus::{GaugeVec, IntGauge};
+use std::collections::hash_map::Entry;
+use std::collections::HashMap;
+use std::sync::Mutex;
+use tracing::warn;
+
+pub(crate) fn export_stats_as_metrics(stats: StoreStatistics) {
+    match ROCKSDB_METRICS.lock().unwrap().export_stats_as_metrics(stats) {
+        Ok(_) => {}
+        Err(err) => {
+            warn!(target:"stats", "Failed to export store statistics: {:?}",err);
+        }
+    }
+}
+
+/// Wrapper for re-exporting RocksDB stats into Prometheus metrics.
+pub(crate) static ROCKSDB_METRICS: Lazy<Mutex<RocksDBMetrics>> =
```

Comment:
Probably can be private.

(I usually don’t care about `pub(crate)`, but for global *state* I do)